lib/web_ui/test/text/layout_service_plain_test.dart

```diff
@@ -41,147 +41,147 @@ void main() {
-  });
-
-  test('preserves whitespace when measuring', () {
-    CanvasParagraph paragraph;
-
-    // leading whitespaces
-    paragraph = plain(ahemStyle, '   abc')..layout(constrain(double.infinity));
-    expect(paragraph.maxIntrinsicWidth, 60);
-    expect(paragraph.minIntrinsicWidth, 30);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l('   abc', 0, 6, hardBreak: true, width: 60.0),
-    ]);
-
-    // trailing whitespaces
-    paragraph = plain(ahemStyle, 'abc   ')..layout(constrain(double.infinity));
-    expect(paragraph.maxIntrinsicWidth, 60);
-    expect(paragraph.minIntrinsicWidth, 30);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l('abc   ', 0, 6, hardBreak: true, width: 30.0),
-    ]);
-
-    // mixed whitespaces
-    paragraph = plain(ahemStyle, '  ab   c  ')
-      ..layout(constrain(double.infinity));
-    expect(paragraph.maxIntrinsicWidth, 100);
-    expect(paragraph.minIntrinsicWidth, 20);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l('  ab   c  ', 0, 10, hardBreak: true, width: 80.0, left: 0.0),
-    ]);
-
-    // single whitespace
-    paragraph = plain(ahemStyle, ' ')..layout(constrain(double.infinity));
-    expect(paragraph.maxIntrinsicWidth, 10);
-    expect(paragraph.minIntrinsicWidth, 0);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l(' ', 0, 1, hardBreak: true, width: 0.0, left: 0.0),
-    ]);
-
-    // whitespace only
-    paragraph = plain(ahemStyle, '     ')..layout(constrain(double.infinity));
-    expect(paragraph.maxIntrinsicWidth, 50);
-    expect(paragraph.minIntrinsicWidth, 0);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l('     ', 0, 5, hardBreak: true, width: 0.0, left: 0.0),
-    ]);
-  });
-
-  test('uses single-line when text can fit without wrapping', () {
-    final CanvasParagraph paragraph = plain(ahemStyle, '12345')
-      ..layout(constrain(50.0));
-
-    // Should fit on a single line.
-    expect(paragraph.alphabeticBaseline, 8);
-    expect(paragraph.maxIntrinsicWidth, 50);
-    expect(paragraph.minIntrinsicWidth, 50);
-    expect(paragraph.width, 50);
-    expect(paragraph.height, 10);
-    expectLines(paragraph, [
-      l('12345', 0, 5, hardBreak: true, width: 50.0, left: 0.0, height: 10.0, baseline: 8.0),
-    ]);
-  });
-
-  test('simple multi-line text', () {
-    final CanvasParagraph paragraph = plain(ahemStyle, 'foo bar baz')
-      ..layout(constrain(70.0));
-    expect(paragraph.alphabeticBaseline, 8);
-    expect(paragraph.maxIntrinsicWidth, 110);
-    expect(paragraph.minIntrinsicWidth, 30);
-    expect(paragraph.width, 70);
-    expect(paragraph.height, 20);
-    expectLines(paragraph, [
-      l('foo bar ', 0, 8, hardBreak: false, width: 70.0, left: 0.0, height: 10.0, baseline: 8.0),
-      l('baz', 8, 11, hardBreak: true, width: 30.0, left: 0.0, height: 10.0, baseline: 18.0),
-    ]);
-  });
+  // test('no text', () {
```

Comment:
What's the plan for these tests?

Reply:
Whoops! They should be fine. I just forgot to uncomment them.